### ISSUE TITLE
Review annotation extensions available on biological process terms

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#2782]
Remove

regulates transcription of
As extension from process branch

Also review other extension prompts on BPs
(we want to encourage that the extension for the entity  is made only on the MF term)

**Comment by kimrutherford** [comment]
>Remove
> regulates transcription of
> As extension from process branch

It's an extension only for GO:0006355 ("regulation of DNA-templated transcription") except for the sub-branch GO:0051090 ("regulation of DNA-binding transcription factor activity").  Is that what needs to be removed?

There are no annotations for GO:0051090 so maybe that has always been a mis-configuration.

**Comment by ValWood** [comment]
yes, remove from
GO:0006355 ("regulation of DNA-templated transcription") 
I can do that but I wan't to check something with you tomorrow, can use this as an example.

**Comment by kimrutherford** [comment]
> yes, remove from
> GO:0006355 ("regulation of DNA-templated transcription")
> I can do that but I wan't to check something with you tomorrow, can use this as an example.

Shall I go ahead and remove it?

**Comment by ValWood** [comment]
Yes please. Eventually I will get to grips with these config files!

**Comment by kimrutherford** [comment]
I've changed the configuration.  I should be gone from Canto by the morning.

**Comment by kimrutherford** [comment]
> I've changed the configuration. It should be gone from Canto by the morning.

That's done now.

> Also review other extension prompts on BPs
> (we want to encourage that the extension for the entity is made only on the MF term)

That's still to do, I think?

**Comment by ValWood** [comment]
Yep, I need a list of all extensions available for process terms. I'll meed to migrate or remove any in use ones.

**Comment by kimrutherford** [comment]
>  Yep, I need a list of all extensions available for process terms.

Do you mean the extensions from the configuration?  I can make a list.

**Comment by ValWood** [comment]
Yes please initially,

**Comment by kimrutherford** [comment]
Here are the extension relations and the number of times that occur in the config:

|extension relation     |count  |
|-----------------------|------:|
|has_input             |12     |
|has_regulation_target|12     |
|regulates_activity_of|12     |
|happens_during        |11     |
|occurs_at             |10     |
|occurs_in             |4      |
|transcript_isoform_id|1      |
|has_qualifier         |1      |
|modified_form         |1      |

This is the BP extension config with some uninteresting columns removed:

|Domain ID                                     |extension relation  |range ID            |Canto display text  |cardinality  |role   |
|----------------------------------------------|--------------------|--------------------|--------------------|-------------|-------|
|GO:0008150                                    |transcript_isoform_id|Text                |transcript isoform ID|0,1          |admin  |
|GO:0008150                                    |has_qualifier      |PBHQ:002            |qualifier           |*           |admin  |
|GO:0008150                                    |modified_form      |Text                |PR:nnnID for modified protein form|*           |admin  |
|GO:0000770                                    |has_input          |SPAPB8E5.05|SPAC513.03|SPCC1795.06|SPBPJ4664.03|exports             |0,1          |user   |
|GO:0006351                                    |occurs_at          |SO:0001865|SO:0001795|occurs at           |0,1          |user   |
|GO:0006396                                    |has_input          |TranscriptID        |processes           |0,1          |user   |
|GO:0006457                                    |has_input          |ProteinID           |folds               |0,1          |user   |
|GO:0006468                                    |has_input          |ProteinID           |phosphorylation of  |0,1          |user   |
|GO:0006473                                    |has_input          |ProteinID           |acetylation of      |0,1          |user   |
|GO:0006486                                    |has_input          |ProteinID           |glycosylation of    |0,1          |user   |
|GO:0006511                                    |has_input          |ProteinID           |involved in degradation of|0,1          |user   |
|GO:0006605                                    |has_input          |ProteinID           |targets             |0,1          |user   |
|GO:0006897                                    |occurs_in          |GO:0032153|GO:0051286|occurs at           |0,1          |user   |
|GO:0007015                                    |occurs_in          |GO:0005826|GO:0051286|occurs at           |0,1          |user   |
|GO:0008150                                    |happens_during     |GO:0022403|GO:0033554|GO:0072690|GO:0044770|GO:0051707|happens during (e.g. cell cycle phase or stress response)|0,1          |user   |
|GO:0016485                                    |has_input          |ProteinID           |processes           |0,1          |user   |
|GO:0016570                                    |occurs_at          |SO:0001798|SO:0001799|occurs at           |0,1          |user   |
|GO:0030702                                    |occurs_at          |SO:0001799          |occurs at           |0,1          |user   |
|GO:0006357                                    |occurs_at          |SO:0001659          |occurs at           |0,1          |user   |
|GO:0031399-is_a(GO:0043666)&is_a(GO:0045859)|has_regulation_target|ProteinID           |regulates modification of|0,1          |user   |
|GO:0033562                                    |has_regulation_target|TranscriptID        |silences            |0,1          |user   |
|GO:0033562                                    |occurs_at          |SO:0000286          |occurs at           |0,1          |user   |
|GO:0006886                                    |has_input          |ProteinID           |transports          |0,1          |user   |
|GO:0034613-is_a(GO:0006886)               |has_input          |ProteinID           |localizes           |0,1          |user   |
|GO:0043487                                    |has_regulation_target|TranscriptID        |stabilizes          |0,1          |user   |
|GO:0043393                                    |has_regulation_target|ProteinID           |regulates binding between (add TWO, i.e. both binding partners)|0,2          |user   |
|GO:0044419                                    |occurs_in          |GO:0110165          |occurs at           |0,1          |user   |
|GO:0045185                                    |has_input          |ProteinID           |maintains location of|0,1          |user   |
|GO:0050684                                    |has_regulation_target|TranscriptID        |regulates processing of|0,1          |user   |
|GO:0050790                                    |regulates_activity_of|ProteinID           |regulates activity of|0,1          |user   |
|GO:0051090                                    |regulates_activity_of|ProteinID           |regulates activity of|0,1          |user   |
|GO:0051223                                    |has_regulation_target|ProteinID           |regulates transport of|0,1          |user   |
|GO:0061013                                    |has_regulation_target|TranscriptID        |regulates catabolism of|0,1          |user   |
|GO:0070613                                    |has_regulation_target|ProteinID           |regulates processing of|0,1          |user   |
|GO:0071168                                    |occurs_at          |SO:0001799          |occurs at           |0,1          |user   |
|GO:0071786                                    |occurs_in          |GO:0005938|GO:0034399|occurs at           |0,1          |user   |
|GO:0090365                                    |has_regulation_target|TranscriptID        |regulates modification of|0,1          |user   |
|GO:0097394                                    |occurs_at          |SO:0001926|SO:0001924|occurs at           |0,1          |user   |
|GO:1901983                                    |has_regulation_target|ProteinID           |regulates acetylation of|0,1          |user   |
|GO:1902983                                    |occurs_at          |SO:0001914|SO:0002021|SO:0001272|occurs at           |0,1          |user   |
|GO:1903533                                    |has_regulation_target|ProteinID           |regulates targeting of|0,1          |user   |
|GO:1903643                                    |occurs_at          |SO:0001900          |occurs at           |0,1          |user   |
|GO:1903827                                    |has_regulation_target|ProteinID           |regulates localization of|0,1          |user   |
|GO:1904595                                    |occurs_at          |SO:0001637|SO:0001272|SO:0001217|occurs at           |0,1          |user   |
|GO:1904616                                    |regulates_activity_of|ProteinID           |regulates activity of|0,1          |user   |

**Comment by ValWood** [comment]
Not too bad! I'll put it on the to do list. This will be good for the new starter if I don't get to it before.  because it supports how we now make GO annotation.
